### ISSUE TITLE
NAN values on results.csv

### ISSUE THREAD

**Issue by AugustoSRodrigues** [ultralytics/ultralytics#8289]
### Search before asking

- [X] I have searched the YOLOv8 [issues](https://github.com/ultralytics/ultralytics/issues) and [discussions](https://github.com/ultralytics/ultralytics/discussions) and found no similar questions.

### Question

Hi, I'm training a detection model on my own database. I'm encountering an issue where after a few epochs, loss values for the validation set become NaN (not a number), and the model stops progressing. When I stop and resume training, the metrics become 0. I would like to know what's causing this.
![nan](https://github.com/ultralytics/ultralytics/assets/57474022/fd1769de-393a-41f0-9de0-7fb95dab21e2)
![metrics](https://github.com/ultralytics/ultralytics/assets/57474022/162d1a83-c5e4-4c4f-a138-6bd7edfe73b8)

**Comment by Kayzwer** [comment]
@AugustoSRodrigues, try to reduce learning rate, and set warmup epochs to 0, the gradient exploded.
